data/sdo-howto-examples.txt

```diff
@@ -303,131 +305,120 @@ JSON:
-                                "duringMedia": "lit-flares.jpg",
-                                "text": "Position your wheel wedges in front of the front tires if a rear tire is flat, or behind the rear tires if a front tire is flat"
-                            },
-                            {
-                                "@type": "HowToTip",
-                                "position": "4",
-                                "text": "You don't want the car to move while you're working on it."
-                            }
-                        ]
-                    }
-                ]
-            },
-            {
-                "@type": "HowToSection",
-                "name": "Raise the car",
-                "position": "2",
-                "steps": {
-                    "@type": "HowToStep",
-                    "itemListElement": [
-                        {
-                            "@type": "HowToDirection",
-                            "position": "1",
-                            "text": "Position the jack underneath the car, next to the flat tire."
-                        },
-                        {
-                            "@type": "HowToDirection",
-                            "position": "2",
-                            "beforeMedia": "car-on-ground.jpg",
-                            "afterMedia": "car-raised.jpg",
-                            "text": "Raise the jack until the flat tire is just barely off of the ground."
-                        },
-                        {
-                            "@type": "HowToTip",
-                            "position": "3",
-                            "text": "It doesn't need to be too high."
-                        },
-                        {
-                            "@type": "HowToDirection",
-                            "position": "4",
-                            "text": "Remove the hubcap and loosen the lug nuts."
-                        },
-                        {
-                            "@type": "HowToDirection",
-                            "position": "5",
-                            "duringMedia": "replacing-tire.mp4",
-                            "text": "Remove the flat tire and put the spare tire on the exposed lug bolts."
-                        },
-                        {
-                            "@type": "HowToDirection",
-                            "position": "6",
-                            "text": "Tighten the lug nuts by hand."
-                        },
-                        {
-                            "@type": "HowToTip",
-                            "position": "7",
-                            "text": "Don't use the wrench just yet."
-                        }
-                    ]
+    "step": [
+        {
+            "@type": "HowToSection",
+            "name": "Preparation",
+            "position": "1",
+            "itemListElement": [
+                {
```

Comment:
Similar to comment above, this would be a list of 2 HowToStep's. 
HowToDirection & HowToTip would be itemListElement nested in the HowToStep